sechub-pds-solutions/gitleaks/tests/copy_sanitycheck_files.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: MIT
+
+current_test_folder="$1"
+	
+if [[ ! -d "$current_test_folder" ]]
+then
+	echo "Target folder is empty"
+	exit 1
+fi
+
```

Comment:
Is the script always called from this directory?
If not, changing into this directory could be added:
```suggestion

cd `dirname $0`
```